app/views/content/subjects/maths/_article.html.erb

```diff
@@ -118,10 +118,13 @@
 
     <h2 class="heading-s">Experience school life</h2>
 
-    <p>If you’re currently doing an undergraduate degree and are interested in teaching maths, you could <a href="/teaching-internship-providers">apply for a paid teaching internship</a>.</p>
-    <p>The 3 week programme could help you to understand what it’s really like in the classroom and get a feel for school life.</p>
-     <p>You can also find out what it's like to be in a classroom through unpaid <a href="/train-to-be-a-teacher/get-school-experience">school experience</a>.</p>
+    <p>If you’re currently doing an undergraduate degree and are interested in teaching maths, you could apply for a paid teaching internship. </p>
+    <p> It could help you to understand what it’s really like in the classroom and get a feel for school life.</p>
+     <p>Internships last for 3 weeks, start in June and you’ll be paid £300 per week while you experience the classroom.</p>
+     <p><a href="/is-teaching-right-for-me/teaching-internship-providers">Find out about teaching internships</a>.
+  </section>
 
+  <section class="col col-space-m col-720">
```

Comment:
Something odd going on with the spacing above the `Improving your maths subject knowledge` heading?